app/src/main/java/com/hedvig/app/feature/offer/OfferItemsBuilder.kt

```diff
@@ -0,0 +1,27 @@
+package com.hedvig.app.feature.offer
+
+import com.hedvig.android.owldroid.graphql.OfferQuery
+import com.hedvig.app.feature.offer.ui.OfferModel
+
+class OfferItemsBuilder {
+
+    companion object {
+        fun createItems(data: OfferQuery.Data): List<OfferModel> {
```

Comment:
I'd be ok with making this a top-level function, or making the `OfferItemsBuilder` an `object`, to get rid of the `companion object` 🙂